.gitignore

```diff
@@ -2,3 +2,5 @@ pkg/*
 **/.DS_Store
 .rspec
 gemfiles
+.ruby-version
```

Comment:
We added it because we did not want to prescribe a ruby version for you.  Should it be 2.3.0 then?